docker/tpu/Dockerfile.cluster

```diff
@@ -0,0 +1,74 @@
+# This dockerfile is used to build the docker image for using Ray to manage TPU slices.
+ARG IMAGE=ghcr.io/stanford-crfm/levanter-base
+ARG TAG=latest
+
+FROM ${IMAGE}:${TAG}
+
+# install docker in docker, but don't start it
+RUN apt-get update && apt-get install -y docker.io
+
+ENV TENSORSTORE_CURL_LOW_SPEED_TIME_SECONDS=60\
+    TENSORSTORE_CURL_LOW_SPEED_LIMIT_BYTES=1024\
+    RAY_USAGE_STATS_ENABLED=0\
+    PATH=/opt/levanter/.venv/bin:$PATH\
+    PYTHONPATH=/opt/levanter:/opt/levanter/src:/opt/levanter/examples:/opt/levanter/tests:src:.\
+    HOME=/home/levanter
+# Install dependencies
+
+RUN apt-get install -y \
+        sudo \
+        git \
+        libjemalloc-dev \
+        wget \
+        cmake \
+        g++ \
+        zlib1g-dev \
+        tmux \
+        screen \
+        rsync \
+        netbase \
+        openssh-client \
+        gnupg
+
+RUN pip install --no-cache-dir \
```

Comment:
Super nit, you could maybe do this as a separate pyproject.toml with all these + the ray dependencies and then `uv sync` here. But definitely can defer to a separate PR.

Reply:
yeah i should go through all my code and switch to uv